ExternalBeamPlanning/Logic/vtkSlicerExternalBeamPlanningModuleLogic.cxx

```diff
@@ -200,7 +200,7 @@ void vtkSlicerExternalBeamPlanningModuleLogic::OnMRMLSceneEndImport()
   // Observe plan events of all plan nodes
   std::vector<vtkMRMLNode*> nodes;
   this->GetMRMLScene()->GetNodesByClass("vtkMRMLRTPlanNode", nodes);
-  for (std::vector<vtkMRMLNode*>::iterator nodeIt=nodes.begin(); nodeIt!=nodes.end(); ++nodeIt)
+  for ( auto nodeIt=nodes.begin(); nodeIt!=nodes.end(); ++nodeIt)

```

Comment:
I see this in many places that there is a space after the opening parenthesis. However, the style guides say otherwise and it makes the style not consistent. Can you please remove the spaces everywhere you added it? ("for ( auto" -> for (auto")